### ISSUE TITLE
Chain link prev hash mismatch error when following user

### ISSUE THREAD

**Issue by nscaife** [keybase/client#23432]
When I attempt to follow user `ewust` I receive the following error:

`▶ ERROR Chain link prev hash mismatch error: Chain mismatch at seqno=12`

Is there a way to resolve this on my end?

my log id: 6d01c84ffee18f3a16cc6c1c

**Comment by maxtaco** [comment]
We'll take a look, thanks for the feedback. It's likely a bug on our side.

**Comment by maxtaco** [comment]
Hah, I think the issue is this:

```      "payload_json": "{\"body\":{\"key\":{\"fingerprint\":\"keybase might not want to allow this here\",\"host\":\"keybase.io\",\"key_id\":\"whatever\",\"uid\":\"okokokokokokok\",\"username\":\"root\"}},\"body\":{\"key\":{\"fingerprint\":\"09c91d5a25079ede05d92585e1f406759dbf4c43\",\"host\":\"keybase.io\",\"key_id\":\"e1f406759dbf4c43\",\"uid\":\"60446d1f121a4d388238a8bff5363500\",\"username\":\"ewust\"},\"service\":{\"name\":\"twitter\",\"username\":\"ewust\"},\"type\":\"web_service_binding\",\"version\":1},\"ctime\":1396555334,\"expire_in\":157680000,\"prev\":\"4ca549cc6e0410cc3c802b43ee8db487e4fbd41262557919135bea7e88d4b262\",\"seqno\":11,\"tag\":\"signature\"}\n",```

Which we probably failed to reject sometime in .... 2014? .. I'm guessing?

**Comment by maxtaco** [comment]
We should probably just deactivate the account since stricter clients are rightly puking on this user's account.

**Comment by nscaife** [comment]
@ewust ☝️

**Comment by ewust** [comment]
Hmm. Is there something I can do from my end to fix things? Happy to delete a tweet or something as needed. I don't recall exactly what I did, but knowing me was probably trying to just poke at things. I meant no harm, honest!

**Comment by maxtaco** [comment]
No worries, we'd likely have to update the client software to make exceptions for these broken links.

**Comment by ewust** [comment]
Aww. I can't delete/update the links myself? And if not, is there anyway I can recover / get back ewust as a username on keybase? Or is that my punishment for tinkering? :p

**Comment by maxtaco** [comment]
No, links are immutable once posted. We'll figure something out, bear with us.

**Comment by nscaife** [comment]
Hey @maxtaco, I get a new error message with 5.4.0:

`▶ ERROR Bad key family: InsertServerEldestKey found a non-matching eldest key.`

**Comment by ewust** [comment]
Any update on this? I am still not able to login with the app:

```
Bad key family: InsertServerEldestKey found a non-matching eldest key.
Error code 218: GENERIC in method keybase.1.login.login
```

Should I give up and make a new account?
